rcl_lifecycle/src/com_interface.c

```diff
@@ -108,6 +108,9 @@ rcl_lifecycle_com_interface_publisher_init(
 
   // initialize publisher
   rcl_publisher_options_t publisher_options = rcl_publisher_get_default_options();
+  publisher_options.qos.reliability = RMW_QOS_POLICY_RELIABILITY_RELIABLE;
```

Comment:
rmw_qos_profile_default already sets this `RMW_QOS_POLICY_RELIABILITY_RELIABLE`, so no need to overwrite this.

Reply:
Thanks, I've removed the line.